package.json

```diff
@@ -24,12 +26,18 @@
     "eslint-config-prettier": "6.10.0",
     "eslint-plugin-prettier": "3.1.2",
     "husky": "4.2.1",
+    "jest-canvas-mock": "^2.2.0",
```

Comment:
pin all dependencies..

Reply:
What do you mean by "pin"?

Reply:
Remove the `^` (range) from the versions and run `npm install` again

Reply:
I solved it but I have a question (mostly based on curiosity). Don't lock files "lock" the dependencies to a specific version; so that, everyone installs the same version?